docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc

```diff
@@ -5,7 +5,9 @@ include::_attributes.adoc[]
 
 With the Quarkus OpenID Connect (OIDC) extension, you can protect application HTTP endpoints by using the OIDC Authorization Code Flow mechanism.
 
-To learn more about the OIDC authorization code flow mechanism, see xref:security-oidc-code-flow-authentication-concept.adoc[OIDC code flow mechanism for protecting web applications] or to learn about other authentication mechanisms, see xref:security-authentication-mechanisms-concept.adoc#other-supported-authentication-mechanisms[authentication mechanisms].
+To learn more about the OIDC authorization code flow mechanism, see xref:security-oidc-code-flow-authentication-concept.adoc[OIDC code flow mechanism for protecting web applications].
+To learn about how well known social providers such as `Google`, `GitHub`, `Microsoft`, `Twitter`, `Apple`, `Facebook` and `Spotify` can be used with Quarkus OIDC, see xref:security-openid-connect-providers.adoc[Well Known OpenID Connect providers].
```

Comment:
Are back-ticks needed?

```suggestion
To learn about how well-known social providers such as Google, GitHub, Microsoft, Twitter, Apple, Facebook, and Spotify can be used with Quarkus OIDC, see xref:security-openid-connect-providers.adoc[Configuring Well-Known OpenID Connect Providers].
```

Reply:
+1 to your comment about the back-ticks. Don't think they are needed here as they are proper nouns and names of providers.